gnocchi/rest/api.py

```diff
@@ -148,18 +148,24 @@ def deserialize(expected_content_types=None):
     return params
 
 
-def validate(schema, data, required=True):
+def validate(schema, data, required=True, detailed_exc=False):
     try:
         return voluptuous.Schema(schema, required=required)(data)
     except voluptuous.Error as e:
```

Comment:
This should only catch `voluptuous.Invalid` so then…

Reply:
done